pkg/audit/reporter.go

```diff
@@ -150,26 +150,26 @@ func (reporter *Reporter) recordOfflineStatus(ctx context.Context, offlineNodeID
 	if len(failed) > 0 {
 		return failed, errs.Combine(Error.New("failed to record some audit offline statuses in overlay"), errlist.Err())
 	}
+
 	return nil, nil
 }
 
 // recordAuditSuccessStatus updates nodeIDs in overlay with isup=true, auditsuccess=true
 func (reporter *Reporter) recordAuditSuccessStatus(ctx context.Context, successNodeIDs storj.NodeIDList) (failed storj.NodeIDList, err error) {
 	defer mon.Task()(&ctx)(&err)
-	var errlist errs.Group
-	for _, nodeID := range successNodeIDs {
-		_, err := reporter.overlay.UpdateStats(ctx, &overlay.UpdateRequest{
+
+	updateRequests := make([]*overlay.UpdateRequest, len(successNodeIDs))
+	for i, nodeID := range successNodeIDs {
```

Comment:
Is there a reason we cannot do just single query for the whole batch?

Reply:
Most of these updates should be possible to condense into a single update.

Reply:
Yes, I agree.  I was looking for opinions as to whether I should put all the business logic in SQL vs using the existing Go logic.  

I will work on the SQL model next.

Reply:
You shouldn't need a SQL model either. An add/subtract with set to DQ should do it.

Reply:
updated to execute updates/deletes in on call to DB